test/unit/event-handler-remove-listener-test.js

```diff
@@ -0,0 +1,42 @@
+const test = require('tap').test
+
+const receiverListener = require('../../event-handler/remove-listener')
```

Comment:
have you meant to call this variable `removeListener`? I think that would make the tests a little more clear!

Reply:
I agree it does look cleaner to call it `removeListener`. I've updated the test with the new name.